clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleBackendServiceCachingAgent.groovy

```diff
@@ -67,14 +67,13 @@ class GoogleBackendServiceCachingAgent extends AbstractGoogleCachingAgent {
     if (globalBackendServices) {
       ret.addAll(globalBackendServices.collect { toGoogleBackendService(it, GoogleBackendService.BackendServiceKind.globalBackendService) })
     }
-    timeExecute(compute.regions().list(project),
-                "compute.regions.list",
-                TAG_SCOPE, SCOPE_GLOBAL
-    ).items.each { Region region ->
+
+    def regions = credentials.regions.collect { it.name }
+    regions.each { Region region ->
```

Comment:
If you've just collected the names of the regions on the line above, this iterator shouldn't define the type as `Region`, right?